packages/server-core/package.json

```diff
@@ -97,7 +97,6 @@
     "lodash": "4.17.21",
     "meshoptimizer": "^0.18.1",
     "mime-types": "^2.1.35",
-    "minio": "7.1.3",
```

Comment:
We dont need to remove this. Because there are some scripts that are using storage provider and if we remove it from here then it will create issues for those scripts.